client/components/domains/register-domain-step/index.jsx

```diff
@@ -1021,7 +1028,7 @@ class RegisterDomainStep extends React.Component {
 				placeholderQuantity={ PAGE_SIZE }
 				isSignupStep={ this.props.isSignupStep }
 				railcarSeed={ this.state.railcarSeed }
-				fetchAlgo={ searchVendor + '/v1' }
+				fetchAlgo={ '/domains/search/' + searchVendor + '/20180816' }
```

Comment:
Could we use `this.props.isSignupStep` to build a string that indicates if the suggestion was shown in signup? It would be useful to be able to distinguish between signup and Calypso.